server_action_navigate/__manifest__.py

```diff
@@ -4,14 +4,15 @@
 
 {
     "name": "Server Actions - Navigate",
-    "version": "13.0.1.0.0",
-    "author": "GRAP, " "Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Ooops, Ashish Hirpara, GRAP, Odoo Community Association (OCA)",
```

Comment:
nitpick : why 2 new authors, for 1 commit ?

Reply:
got it, updated. Thanks.